py/tests/encoders/date_test.py

```diff
@@ -77,19 +78,45 @@ def testDayOfWeek(self):
     # In the middle of fall, Thursday, not a weekend, afternoon - 4th Nov,
     # 2010, 14:55
     d = datetime.datetime(2010, 11, 4, 14, 55)
-    # d = datetime.datetime(2010, 11, 1, 8, 55) # DEBUG
-    # Any Monday morning (before noon) # DEBUG
-    # d = datetime.strptime("22/07/19 8:00", "%d/%m/%y %H:%M") # DEBUG
     bits = enc.encode(d)
 
-    # Week is MTWTFSS contrary to localtime documentation, Monday = 0 (for
-    # python datetime.datetime.timetuple()
+    # Week is MTWTFSS, 
+    # Monday = 0 (for python datetime.datetime.timetuple())
     dayOfWeekExpected = [0,0,0,1,0,0,0] #Thu
```

Comment:
Thursday is now again back to being 4th day of week in the representation, which is nice.